Airports/EGDR/SMR/GEO.txt

```diff
@@ -0,0 +1,18 @@
+N050.05.09.316 W005.15.51.396 N050.05.09.686 W005.15.50.709 smrYellow ;Hold_F2
```

Comment:
```suggestion
N050.05.09.316 W005.15.51.396 N050.05.09.686 W005.15.50.709 standHold ;Hold_F2
```
Could you change the hold colour to the same as above to keep it the same across all the mil SMR's please